hypervisor/include/debug/profiling_internal.h

```diff
@@ -21,6 +21,7 @@
 #define SEP_BUF_ENTRY_SIZE 		32U
 #define SOCWATCH_MSR_OP			100U
 
+#define MAGIC_NUMBER			99999988U
```

Comment:
usually people uses hex # for magic. What is the purpose of decimal here?

Reply:
anyway, hex is better for debug

Reply:
Updated to HEX